vendor/k8s.io/kubernetes/pkg/kubectl/cmd/patch.go

```diff
@@ -153,7 +153,7 @@ func (o *PatchOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []st
 	}
 
 	o.namespace, o.enforceNamespace, err = f.ToRawKubeConfigLoader().Namespace()
-	if err != nil {
+	if err != nil && !o.Local {
```

Comment:
@deads2k This error needs to be ignored, because otherwise this command will fail in `hack/lib/start.sh` due to an incomplete kubeconfig (basically it needs to be able to bootstrap a config without also depending on it)